src/Handlebars.Net.Helpers/Helpers/BooleanHelpers.cs

```diff
@@ -0,0 +1,29 @@
+﻿using HandlebarsDotNet.Helpers.Attributes;
+using HandlebarsDotNet.Helpers.Enums;
+
+namespace HandlebarsDotNet.Helpers.Helpers;
+
+internal class BooleanHelpers : BaseHelpers, IHelpers
+{
+    public BooleanHelpers(IHandlebars context) : base(context)
```

Comment:
Should we also add all boolean operators?

Like "Xor" , "Or", and so on?

Reply:
Not a bad idea. I mostly added this as a need for `Not`, but we can certainly expand.

Reply:
I added `Or`, `LogicalOr`, `LogicalXor`, `And`, and `LogicalAnd`.

See https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/operators/boolean-logical-operators